docs-content/general/6_product-features/1_session-replay/ignoring-sessions.md

```diff
@@ -10,19 +10,21 @@ updatedAt: 2022-06-27T03:34:47.000Z
 
 ## Ignore sessions by user identifier
 In some cases, you may want to ignore sessions from a specific user. You can do this by adding the user identifier to the "Excluded Sessions" input under the "Session Replay" tab in your [project settings](https://app.highlight.io/settings). Please note that we use the `identifier` (or first argument) sent in your `H.identify` method to ignore against (SDK docs [here](../../../sdk/client.md)).
+## Ignoring sessions without an error
```

Comment:
Added to this [page](http://localhost:4000/docs/general/product-features/session-replay/ignoring-sessions):

![Screenshot 2023-05-01 at 10 25 04 AM](https://user-images.githubusercontent.com/58678/235487488-196efb4b-6a75-4a4a-b6f8-a15dd99c0b70.png)